app/src/main/java/at/roteskreuz/stopcorona/screens/reporting/reportStatus/ReportingStatusViewModel.kt

```diff
@@ -60,8 +48,8 @@ class ReportingStatusViewModel(
                     uploadReportDataStateObserver.idle()
                     return@launch
                 }
-                val temporaryTracingKeys = exposureNotificationRepository.getTemporaryExposureKeys()
```

Comment:
I don´t like this beeing removed here. The exception handling is tightly related to the `getTemporaryExposureKeys()` calls and moving it inside of the function could lead to non handling of it in other situations

Reply:
Also, dupplication of code

Reply:
The exception handling doesn't need to be tightly related based on the proximity of the code. In this case the ApiException triggered in the uploadReportInformation() will be propagated upwards and will be still caught in the try/catch of the ViewModel. 

I've updated the KDoc for `uploadReportInformation` to mention that the ApiException is thrown.

I would say the current use of `getTemporaryExposureKeys()` in ReportingRepository is not an example of duplication of code.

Reply:
After develop is merged here, the exposureNotificationManager checks the running state. This should be also moved to the uploadReport method.